docs/src/main/sphinx/release/release-393.md

```diff
@@ -35,17 +42,28 @@
 ## Hive connector
 
 * Add support for {doc}`/sql/merge`. ({issue}`7933`)
+* Add support for bucket filtering on bucketed columns of `float`, `double`,
+  `date`, `list`, `map` and `bounded varchar` data types. ({issue}`13553`)
+* Upgrade Alluxio to version 2.8.1 to fix related security vulnerabilities. ({issue}`13609`)
 * Fix bug where query constraints were not used when fetching table statistics. ({issue}`13488`)
 * Fix error when writing to a table with only `date`-typed columns while using
   the Hive metastore. ({issue}`13502`)
 
 ## Iceberg connector
 
 * Add support for {doc}`/sql/merge`. ({issue}`7933`)
+* Add support for filtering splits based on `$file_modified_time` column
+  predicates. ({issue}`13504`)
 * Improve performance of read queries on Iceberg v2 tables with
   deletion-tracking files. ({issue}`13395`)
 * Allow partitioning over columns which use whitespace in their names. ({issue}`12226`)
 * Fix error when querying tables which are empty and contain no table history. ({issue}`13576`)
+* Fix deletion queries sometimes truncating the table history in the
+  `$snapshots` system table. ({issue}`12843`)
+* Fix query failure when optimizing an Iceberg table which is empty and contains
+  no table history. ({issue}`13582`)
+* Fix incorrect query results when reading from a stale materialized view that
```

Comment:
can we drop "stale" ? Or reword .. also ... if the MV is stale .. doesnt it just read from the underlying tables cleanly?